service/src/main/java/uk/gov/hmcts/reform/fpl/service/ChildrenService.java

```diff
@@ -85,4 +86,20 @@ private void addHiddenValues(Element<Child> element, Child.ChildBuilder builder)
             .partyType(INDIVIDUAL)
             .build());
     }
+
+    // TODO: 31/01/2020 test me
+    public String getChildrenLabel(List<Child> children) {
+        if (isEmpty(children)) {
+            return "No children in the case";
+        }
+
+        StringBuilder builder = new StringBuilder();
+
+        for (int i = 0; i < children.size(); i++) {
+            final String name = children.get(i).getParty().getFullName();
+            builder.append(String.format("Child %d: %s%n", i + 1, name));
+        }
+
+        return builder.toString();
+    }
```

Comment:
this would be lovely to make generic, I don't think we can though 😢

Reply:
Not that I can think of, best we could do is pass a function as a parameter that does the conversion to a string.
Something like
```java
public <T> String getLabel(List<T> objects, Function<T, String> converter)
```